test/functionalTest/cases/3190_notify_only_attributes_that_change/notify_only_attributes_that_change_erros.test

```diff
@@ -0,0 +1,187 @@
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Notify only attributes that change for NGSIv2 for error POST /v2/subscriptions, POST /v2/subscriptions, POST /v2/subscriptions.
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print localhost $LISTENER_PORT
+
+--SHELL--
+
+#
+# 01. POST /v2/subscriptions, create subscription with attrs, exceptAttrs and onlyChangedAttrs.
+# 02. POST /v2/subscriptions, create subscription with onlyChangedAttrs.
+# 03. POST /v2/subscriptions, create subscription with attrs and onlyChangedAttrs set to incorrect value.
+#
+
+echo '01. POST /v2/subscriptions, create subscription with attrs, exceptAttrs and onlyChangedAttrs.'
+echo '============================================================================================='
+payload='{
+  "description": "A subscription to get info about Room1",
+  "subject": {
+    "entities": [
+      {
+        "id": "Room1",
+        "type": "Room"
+      }
+    ],
+    "condition": {
+      "attrs": [
+        "A","B","C"
+      ]
+    }
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify"
+    },
+	"attrs" : [
+	  "A","B"
+	],
+	"exceptAttrs" : [
+	  "C"
+	],
+    "onlyChangedAttrs":true
+  },
+  "expires": "2040-01-01T14:00:00.00Z",
+  "throttling": 5
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+echo
+echo
+
+
+echo '02. POST /v2/subscriptions, create subscription with onlyChangedAttrs.'
+echo '======================================================================'
+payload='{
+  "description": "A subscription to get info about Room1",
+  "subject": {
+    "entities": [
+      {
+        "id": "Room1",
+        "type": "Room"
+      }
+    ],
+    "condition": {
+      "attrs": [
+        "A","B","C"
+      ]
+    }
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify"
+    },
+    "onlyChangedAttrs":true
+  },
+  "expires": "2040-01-01T14:00:00.00Z",
+  "throttling": 5
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+echo
+echo
+
+
+echo '03. POST /v2/subscriptions, create subscription with attrs and onlyChangedAttrs set to incorrect value.'
+echo '======================================================================================================='
+payload='{
+  "description": "A subscription to get info about Room1",
+  "subject": {
+    "entities": [
+      {
+        "id": "Room1",
+        "type": "Room"
+      }
+    ],
+    "condition": {
+      "attrs": [
+        "A","B","C"
+      ]
+    }
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify"
+    },
+	"attrs" : [
+	  "A","B"
+	],
+	"onlyChangedAttrs": trrue
```

Comment:
Instead of this error test (which is already pretty covered, as any other JSON syntax error in general) I'd suggest to test what happens if user tries to set "onlyChangedAttrs" to a non bolean value. For instance:

```
"onlyChagnedAttrs": "foo"
```

Reply:
fixed in f9887f1